### ISSUE TITLE
Support for aarch64 docker hosts

### ISSUE THREAD

**Issue by qstorey** [Joxit/docker-registry-ui#162]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
First off, thank you very much for building docker-registry-ui. I have been using it for the last two years and really happy with it.
I recently tried to run it on a docker host with an aarch64 CPU but the container failed to start. I have noticed on the [Docker Hub project](https://hub.docker.com/r/joxit/docker-registry-ui/tags?page=1&ordering=last_updated) there are tags appearing with support for ARM but these are not with the tagged releases of docker-registry-ui (1.5.0, 1.5.1, etc).
Could you please release Docker images for aarch64 as well?

**Describe alternatives you've considered**
n/a

**Additional context**
```bash
root@host:~ docker run --rm -ti joxit/docker-registry-ui:1.5
standard_init_linux.go:211: exec user process caused "exec format error"

root@host:~ uname -a
Linux host 5.4.0-1022-raspi #25-Ubuntu SMP PREEMPT Thu Oct 15 13:31:49 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux

root@host:~ docker -v
Docker version 19.03.13, build 4484c46
```

**Comment by Joxit** [comment]
Hi, thank you for your issue and using my project :smile: 

The image for aarch64 should be `joxit/docker-registry-ui:arm64v8` but yes, this will refer to the latest version. It was a bit complicated to manage versions for all architectures... :confused: But now, with [github action](https://github.com/marketplace/actions/build-and-push-docker-images), maybe I can do something

**Comment by qstorey** [comment]
Hopefully it should be possible with GitHub actions, but I will give the `arm64v8` tag a try for now. Thanks.

**Comment by Joxit** [comment]
Hi, I tried something with github actions.
Can you tell me if the image `joxit/docker-registry-ui:master-static` is working for you ?

**Comment by qstorey** [comment]
Works 🎉 
```bash
root@host:~ docker run --rm -ti joxit/docker-registry-ui:master-static
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf differs from the packaged version
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/90-docker-registry-ui.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
```

I was able to browse to the IP & port and the HTML rendered, however getting it to connect to the my registry's API didn't work despite setting the `REGISTRY_URL` environment variable. But in my normal setup I don't use the static image you provide, so maybe it has something to do with that.

So most importantly the image is able to run on my ARM host. Thanks for getting around to this so quickly.

**Comment by qstorey** [comment]
I see you released 1.5 recently with support for multiple architectures so I'm going to close the issue. Thanks!
